R/Constants.R

```diff
@@ -66,6 +66,12 @@ sos200_emptyGetObservationResponseString <-
          "<sos:GetObservationResponse xmlns:sos=\"http://www.opengis.net/sos/2.0\" ",
          "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" ",
          "xsi:schemaLocation=\"http://www.opengis.net/sos/2.0 http://schemas.opengis.net/sos/2.0/sosGetObservation.xsd\"/>")
+sos200_emptyGetDataAvailabilityResponseString <-
```

Comment:
Please replace with `xml2` - based parsing of the response, and then probably just see if `xml2::xml_text(..)` is empty. If I understand this correctly, this will break if there is a different namespace used..

Same for `sos200_emptyGetObservationResponseString`.